editors/code/src/commands.ts

```diff
@@ -336,7 +336,7 @@ export function toggleInlayHints(_ctx: Ctx): Cmd {
 // The contents of the file come from the `TextDocumentContentProvider`
 export function syntaxTree(ctx: Ctx): Cmd {
     const tdcp = new (class implements vscode.TextDocumentContentProvider {
-        readonly uri = vscode.Uri.parse("rust-analyzer://syntaxtree/tree.rast");
+        readonly uri = vscode.Uri.parse("rust-analyzer-syntax-tree://syntaxtree/tree.rast");
```

Comment:
Maybe change the uri's into something like this?
```suggestion
        readonly uri = vscode.Uri.parse("rust-analyzer-syntax-tree://tree.rast");
```

Reply:
I think we should go the other way around and have a single scheme, but sure.

Reply:
For some reason this breaks the tab title (it shows `.` instead of `tree.rast` or whatever).